database/src/main/resources/db/migration/V1.57.0__NOTIF-603_anemic_tenants.sql

```diff
@@ -0,0 +1,26 @@
+-- Table: behavior_group
+DROP INDEX ix_behavior_group_account_id;
+
+-- Table: email_aggregation
+ALTER TABLE email_aggregation
+    ALTER COLUMN account_id DROP NOT NULL,
+    ALTER COLUMN org_id SET NOT NULL;
+
+-- Table: endpoint_email_subscriptions
+ALTER TABLE endpoint_email_subscriptions
+    DROP CONSTRAINT pk_endpoint_email_subscriptions,
+    DROP CONSTRAINT uq_endpoint_email_subscriptions,
+    ALTER COLUMN account_id DROP NOT NULL,
+    ALTER COLUMN org_id SET NOT NULL,
+    ADD CONSTRAINT pk_endpoint_email_subscriptions PRIMARY KEY (org_id, user_id, subscription_type, application_id);
+
+-- Table: endpoints
+DROP INDEX ix_endpoints_account_id;
+
+-- Table: event
+DROP INDEX ix_event_account_id;
+DROP INDEX ix_event_account_id_application_id;
+DROP INDEX ix_event_index_only_scan;
+ALTER TABLE event
+    ALTER COLUMN account_id DROP NOT NULL,
+    ALTER COLUMN org_id SET NOT NULL;
```

Comment:
This line won't work on prod for now. The DB contains old events that were not migrated because of an empty `account_id` field. They will be automatically purged eventually, but we could also purge them sooner manually.

Reply:
Are these events even valid? 
if not, we could clean them with this migration

Reply:
No, they're not valid. One of the onboarded apps sent an empty `account_id` field by mistake for a while.

I know that Flyway sometimes has difficulties with migration scripts that contain both DML and DDL. I can't say for sure that it will work if we removing the events from this script.

Reply:
I'll check later for Flyway :smiley:

Reply:
ack - could we create a new migration script for that then? or when are these events up for automatic cleaning?

Reply:
Yes, we can add another DML script to this PR with a version lower than the DDL script.

I think they won't be automatically purged before one or maybe two weeks.

Reply:
Done.